container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -906,13 +958,102 @@ instance_groups:
                   topologyKey: "beta.kubernetes.io/os"
   - name: statsd_injector
     release: statsd-injector
+  - name: bpm
+    release: bpm
+    properties:
+      bosh_containerization:
+        run:
+          capabilities: [ALL]
+          service-account: privileged
+- name: log-cache-scheduler
+  scripts:
+  - scripts/forward_logfiles.sh
+  - scripts/patches/fix_monit_rsyslog.sh
+  - scripts/patches/fix_logcache_certs_forwarder.sh
+  - scripts/patches/fix_logcache_certs_scheduler.sh
+  jobs:
+  - name: global-properties # needs to be first so images use it for processing monit templates
+    release: scf-helper
+  - name: log-cache-scheduler-properties
+    release: scf
+  - name: authorize-internal-ca
+    release: scf
+  - name: log-cache-scheduler
+    release: log-cache
+    # defaults
+  - name: log-cache-expvar-forwarder
+    release: log-cache
+    # defaults
+    properties:
+      bosh_containerization:
+        colocated_containers:
+        - loggregator-agent
+        run:
+          service-account: privileged
+          scaling:
+            min: 1
+            max: 65535
+            ha: 2
+          capabilities: []
+          memory: 410
+          virtual-cpus: 2
+          affinity:
+            podAntiAffinity:
+              preferredDuringSchedulingIgnoredDuringExecution:
+              - weight: 100
+                podAffinityTerm:
+                  labelSelector:
+                    matchExpressions:
+                    - key: "app.kubernetes.io/component"
+                      operator: In
+                      values:
+                      - doppler
+                  topologyKey: "beta.kubernetes.io/os"
+  configuration:
+    templates:
+      properties.tls.ca_cert: '"((INTERNAL_CA_CERT))"'
+      properties.tls.cert: '"((LOG_CACHE_CERT))"'
+      properties.tls.key: '"((LOG_CACHE_CERT_KEY))"'
+      properties.maps: >
+        [{ "name": "worker-assignments",
+	   "source_id": "log-cache-scheduler",
+	   "addr": "http://localhost:6064/debug/vars",
+	   "template": "{{.WorkerAssignments | jsonMap}}" },
+	 { "name": "worker-health",
+	   "source_id": "log-cache-scheduler",
+	   "addr": "http://localhost:6064/debug/vars",
+	   "template": "{{.WorkerHealth | jsonMap}}" }
+	]
```

Comment:
This will probably break (because YAML hates tabs).

Reply:
Oops. Sure, will fix.

Reply:
Done with commit 35498d43eecd091908c6e7c4852e10b0509937bb